os/board/rtl8730e/src/component/usrcfg/rtl8730e/ameba_pinmapcfg.c

```diff
@@ -27,78 +27,78 @@
-	{_PA_17,			GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN},			 //
-	{_PA_18,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PA_19,			GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN},			 //
-	{_PA_20,			GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN},			 //
-	{_PA_21,			GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN},			 //
-	{_PA_22,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PA_23,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PA_24,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PA_25,			GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN},			 //
-	{_PA_26,			GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN},			 //
-	{_PA_27,			GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN},			 //
-	{_PA_28,			GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN},			 //
-	{_PA_29,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PA_30,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PA_31,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PB_0,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PB_1,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PB_2,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PB_3,			GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN},			 //
-	{_PB_4,			GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN},			 //
-	{_PB_5,			GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN},			 //
-	{_PB_6,			GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN},			 //
-	{_PB_7,			GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN},			 //
-	{_PB_8,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PB_9,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PB_10,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PB_11,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PB_12,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PB_13,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PB_14,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PB_15,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PB_16,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PB_17,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PB_18,			GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN},			 //
-	{_PB_19,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PB_20,			GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN},			 //
-	{_PB_21,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 // boot sel
-	{_PB_22,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PB_23,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PB_24,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 // PB24 internal pull high(80k)
-	{_PB_25,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PB_26,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PB_27,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PB_28,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PB_29,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PB_30,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PB_31,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //
-	{_PC_0,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PC_1,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PC_2,			GPIO_PuPd_UP,				GPIO_PuPd_DOWN},			 //flash clk pull down for PG or 50uA
-	{_PC_3,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PC_4,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PC_5,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PC_6,			GPIO_PuPd_UP,				GPIO_PuPd_UP},			 //
-	{_PNC,			GPIO_PuPd_KEEP,				GPIO_PuPd_KEEP},			 //table end
+	//Pin Name		Func PU/PD			Slp PU/PD
```

Comment:
This file still shows every line is modified in the first commit. Could you leave real change in the first?